packages/rainbowkit/src/components/ConnectOptions/DesktopOptions.css.ts

```diff
@@ -18,12 +18,7 @@ export const ScrollClassName = style([
 
 // biome-ignore format: design system keys
 export const sidebar = style({
-  '@media': {
-    [`screen and (min-width: ${largeScreenMinWidth}px)`]: {
-      minWidth: '287px',
-    },
-  },
-  'minWidth': '246px',
+  minWidth: '287px',
```

Comment:
Sidebar has weird `minWidth` change from `246px` to `287px` so made sure it's `287px` only 👍